ui-tests/src/test/java/io/syndesis/qe/steps/integrations/IntegrationSteps.java

```diff
@@ -136,7 +136,7 @@ public void expectIntegrationNotPresent(String name) {
     public void waitForIntegrationState(String integrationName, String integrationStatus) {
         SelenideElement integration = listPage.getListComponent().getIntegration(integrationName);
         assertTrue(TestUtils.waitForEvent(status -> status.equals(integrationStatus), () -> listPage.getListComponent().getIntegrationItemStatus(integration),
-                TimeUnit.MINUTES, 5, TimeUnit.SECONDS, 5));
+                TimeUnit.MINUTES, 1, TimeUnit.SECONDS, 1));
```

Comment:
Any reason to change wait timeout to 1 min, it's too shot to get into Published?